src/docker.test.ts

```diff
@@ -264,10 +288,14 @@ describe("Docker images", (): void => {
     },
     {
       examples: [
-        ["my-key", false, false, [["preexisting-image"], ["new-image"]]],
-        ["my-key", false, false, [["preexisting-image"], []]],
-        ["my-key", false, true, [["preexisting-image"], ["new-image"]]],
-        ["my-key", true, false, [["preexisting-image"], ["new-image"]]],
+        ["my-key", false, false, false, [["preexisting-image"], ["new-image"]]],
+        ["my-key", false, false, false, [["preexisting-image"], []]],
+        ["my-key", false, true, false, [["preexisting-image"], ["new-image"]]],
+        ["my-key", true, false, false, [["preexisting-image"], ["new-image"]]],
+        ["my-key", false, false, true, [["preexisting-image"], ["new-image"]]],
```

Comment:
This is the only new example that is needed to ensure test coverage. The examples aren't used to cover all cases since fast-check is really good at that. They are just used to ensure that every line of production code gets hit.